src/main/python/TravelTimeReporter.py

```diff
@@ -126,6 +127,52 @@ def read_active_skims(self):
                 ["i", "j"]
             )
 
+        # Read bike and walk times from AM traffic skim
+        am_traffic_skim_file = os.path.join(
+            self.model_run,
+            "output",
+            "skims",
+            "traffic_skims_AM.omx"
+        )
+        skims = omx.open_file(am_traffic_skim_file, "r")
+        zones = skims.mapping("zone_number").keys()
+        active_taz_skims = {
+            "BIKE_TIME": "taz_bike_time",
+            "walkTime": "taz_walk_time"
+        }
+        for core in active_taz_skims:
+            skim_values = np.array(skims[core.format(self.settings["time_period"])])
```

Comment:
@JoeJimFlood could you please explain why the ".format" in "core.format(self.settings["time_period"])" is needed?

Reply:
The [matrix names](https://github.com/SANDAG/ABM/blob/ABM3_develop/src/main/python/AMTravelTimeReporterConfigs.yaml#L7) in that are read in from the config files have `{}` as a placeholder for the time period. `str.format()` replaces the brackets with whatever is in the function, so in this case '{}` will become AM or MD depending on the time period.

As an example:
```
"WALK_LOC_XFERWALK__{}".format("AM")
```
becomes:
```
"WALK_LOC_XFERWALK__AM"
```

Reply:
Thanks for removing .format since it is unnecessary for active taz skims.